pkg/views/styles.go

```diff
@@ -24,6 +29,7 @@ var (
 	Gray        = lipgloss.AdaptiveColor{Light: "243", Dark: "243"}
 	LightGray   = lipgloss.AdaptiveColor{Light: "#828282", Dark: "#828282"}
 	Red         = lipgloss.AdaptiveColor{Light: "#FF4672", Dark: "#ED567A"}
+	Purple      = lipgloss.AdaptiveColor{Light: "#800080", Dark: "#800080"}
```

Comment:
No need to add a new Purple style. `Load more` should be green as well.

Reply:
purple is just to make the 'load more' option little distinctive from the other choices, comment if we should still remove..

Reply:
We should yes since we don't have the purple anywhere. You can make the `Load more` text bold so it's more visible.

Reply:
done